### PR TITLE
gitignore tmp files

### PR DESCRIPTION
**Pull request by canonici**

### CODE REVIEW

File: gwimages/gwimages.ml [hunk 1]
```diff
@@ -0,0 +1,47 @@
+
```
Comment (ilankri) [inline gwimages/gwimages.ml]:
```suggestion
```

File: gwimages/gwimages.ml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    Lock.control_retry (Files.lock_file bname) ~onerror:Lock.print_error_and_exit @@ fun () ->
+    let base = Gwdb.open_base bname in
+    let oc = open_out output_file in
+    output_images_of_base oc base;
+    close_out oc;
+    Gwdb.close_base base
+  | _ ->
+    Arg.usage speclist usage ;
+    exit 2;
+
```
Comment (ilankri) [inline gwimages/gwimages.ml]:
```suggestion
```

File: gwimages/gwimages.ml [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+let key_of_person base p =
+  let surname = Name.lower (Gwdb.p_surname base p) in
+  let first_name = Name.lower (Gwdb.p_first_name base p) in
+  let occ = string_of_int (Gwdb.get_occ p) in
+  let key = String.concat "." [surname; first_name; occ] in
+  key
```
Comment (ilankri) [inline gwimages/gwimages.ml]:
```suggestion
  String.concat "." [surname; first_name; occ]
```

File: gwimages/gwimages.ml [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+    Secure.set_base_dir (Filename.dirname bname);
+    Lock.control_retry (Files.lock_file bname) ~onerror:Lock.print_error_and_exit @@ fun () ->
+    let base = Gwdb.open_base bname in
+    let oc = open_out output_file in
+    output_images_of_base oc base;
+    close_out oc;
+    Gwdb.close_base base
+  | _ ->
+    Arg.usage speclist usage ;
+    exit 2;
```
Comment (ilankri) [inline gwimages/gwimages.ml]:
```suggestion
  if !bname <> "", !output_file <> "" then
    Secure.set_base_dir (Filename.dirname !bname);
    Lock.control_retry (Files.lock_file !bname) ~onerror:Lock.print_error_and_exit @@ fun () ->
    let base = Gwdb.open_base !bname in
    let oc = open_out !output_file in
    output_images_of_base oc base;
    close_out oc;
    Gwdb.close_base base
  else (
    Arg.usage speclist usage;
    exit 2
  )
```

File: gwimages/gwimages.ml (hunk 3 above)
Comment (hgouraud) [reply gwimages/gwimages.ml]: Today, portrait files are named first_name.oc.surname.ext
Are we departing from this format?

Comment (canonici) [reply gwimages/gwimages.ml]: it is not the only way you can reference a portrait file, you can also use an external url. We are exporting these.

File: gwimages/dune~
Comment (ilankri) [inline gwimages/dune~:1]: To be removed.

File: gwimages/gwimages.ml~
Comment (ilankri) [inline gwimages/gwimages.ml~:1]: To be removed.

**Review by ilankri — APPROVED** [review]

**Pull request merged**